_styles/main.css

```diff
@@ -268,14 +268,49 @@ html.ie-legacy .nav-content > ul.right {
   float: right;
 }
 
+nav .menu-button {
+  display: none;
+  align-items: center;
+  justify-content: center;
+  flex: none;
+  width: 24px;
+  height: 24px;
+  padding: 0;
+  margin: 12px 24px;
+  border: none;
+}
+
+nav .menu-button.enabled {
+  display: inline-block;
+}
+
+nav .menu-button.enabled + .nav-content {
+  display: none;
+}
+
+nav .menu-button.enabled[aria-expanded='true'] +  .nav-content {
+  display: flex;
+}
+
+nav .menu-button,
+nav .menu-button:hover,
+nav .menu-button:focus,
+nav .menu-button:active {
+  background-color: none;
```

Comment:
`none` is not a valid property for background color. What were you trying to accomplish with this value here? I notice that the background of the SVG is slightly off of the color of the rest of the page.

Reply:
I intended to remove the menu button's default `background` styling it. That line should say: `background: none`.